src/components/app/DocumentList.js

```diff
@@ -247,6 +251,27 @@ class DocumentList extends Component {
         }
     }
 
+    /**
+     * get supportAttribute of the selected row from the table
+     */
+    getSupportAttribute = (props) => {
```

Comment:
pls rename this method to something like loadSupportAttributeFlag because the name as it is now it's misleading.
Usually, when reading "get" the developer thinks that the method will return a value and it will NOT modify object's internal state. Which in this case it's not true.